lab-adam/route/hospital-route.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const debug = require('debug')('hospital:hospital-route');
+
+const createError = require('http-errors');
+const City = require('../model/city.js');
+const Hospital = require('../model/hospital.js');
+
+const hospitalRouter = module.exports = new Router();
+
+hospitalRouter.get('/api/city/:cityID/hospital/:id', function(req, res, next){
```

Comment:
There is no need to require the client to pass in the city ID as well if you're just trying to get a hospital as you're not concerned with the city for this request.